packages/paste-core/components/typography/src/index.tsx

```diff
@@ -0,0 +1,3 @@
+import {Paragraph} from './Paragraph';
```

Comment:
Looks good! 
You can shorthand this as `export {Paragraph} from './Paragraph'`
Also let's make it a rule to always export the props for every component too
 `export {Paragraph, ParagraphProps} from './Paragraph'`

Reply:
Changed the exports here.

I kept types in a separate file for now so we can place all typography types there for Heading, Paragraph, and Lists.